templates/documentation/cmip6_downscaled.html

```diff
@@ -0,0 +1,149 @@
+{% extends 'base.html' %} {% block content %}
+<h2>CMIP6, Downscaled</h2>
+
+<p>
+  This endpoint provides access to daily CMIP6 data that have been bias-adjusted and downscaled to 4km using ERA5
+  reference data. Select models and scenarios are available. Read more about our CMIP6 subset
+  <a href="https://arcticdatascience.org/item/story-arctic-climate-data-node">here</a>. This dataset spans years
+  1965&ndash;2100 and its extent covers most of Alaska and parts of western Canada. A 6-model average is also available,
+  consisting of an equal-weighted mean of the KACE-1-0-G, MIROC6, MPI-ESM1-2-HR, MRI-ESM2-0, NorESM2-MM, and TaiESM1
+  models.
+</p>
+
+<h3>Service endpoints</h3>
+
+<h4>CMIP6 downscaled daily data</h4>
+
+<p>
+  Query data for provided models, scenarios, and variables. These variables are defined <a
+    href="#cmip6-variable-information">below</a>.
+</p>
+
+<table class="endpoints">
+  <thead>
+    <tr>
+      <th class="endpoint-label">Endpoint</th>
+      <th class="endpoint-url">Example URL</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Daily values at a given latitude and longitude</td>
+      <td style="white-space: nowrap;">
+        <a href="/cmip6_downscaled/point/61.5/-147?models=6ModelAvg&scenarios=ssp585&vars=tasmax">
+          /cmip6_downscaled/point/61.5/-147?models=6ModelAvg&scenarios=ssp585&vars=tasmax</a>
+      </td>
+    </tr>
+  </tbody>
+  <tfoot>
+    <tr>
+      <td colspan="2">
+        Multiple models can be requested by appending a comma-separated string
+        like <code>?models=&lt;model_1&gt;,&lt;model_2&gt;</code>.<br />
+        Multiple scenarios can be requested by appending a comma-separated
+        string like <code>?scenarios=&lt;scenario_1&gt;,&lt;scenario_2&gt;</code>.<br />
+        Multiple variables can be requested by appending a comma-separated
+        string like <code>?vars=&lt;var_id_1&gt;,&lt;var_id_2&gt;</code>.<br />
+        CSV output is also available by appending <code>?format=csv</code> to
```

Comment:
Add sentence(s) somewhere about how to get all vars and/or all models and/or all scenarios (i.e. seems like this happens by default when query params are omitted) which I think is a sane default: you don't know which model(s) you want? Then you get all of them.